### DISCUSSION TITLE
[New Device Support] Waipu TV Stick

### DISCUSSION THREAD

**Discussion by tomg1970** [PRProd/HA-Firemote#75] [New Device Family Support Request]
Hello, is it possible to include the waipu tv stick in the integration.

This is a very successful Android tv stick

That would be very great

**Comment by PRProd** [comment]
@tomg1970 - Thanks for your request!  I'm always looking ahead to see what I should add next.  Those discussions are happening at #41.  Please add your request over there, and it will be considered for the next update.

**Comment by PRProd** [comment]
@tomg1970 - Moved to a discussion. I will reply more to this thread if/when I begin official support work for the Waipu TV Stick

**Reply by tomg1970** [reply]
Thanks

**Comment by PRProd** [comment]
The poll over at https://github.com/PRProd/HA-Firemote/discussions/41 has been reset once again after the latest release which added Chromecast support, which means that your previous vote has been removed.  If you're still interested in support for this one, I recommend going back and voting again.

**Comment by AlexZ1402** [comment]
waipu TV doesn't work and you can't select it

**Reply by PRProd** [reply]
Hello @AlexZ1402!

You are correct.  This is because the Waipu TV Stick is not supported yet.  If you haven't voted for it already, please click over to #41 and cast your vote for what you would like to see added to Firemote next.

Please also understand that it is difficult to get one of those shipped to me in the US without a lot of added cost.  I cannot add it to the Firemote card properly until I have one that I can use for testing.

If your vote can swing me towards the Waipu TV Stick next, and/or if you are able to help me get one, that would really help me move forward. 

-Doug

**Reply by AlexZ1402** [reply]
ok, that confuses me because waiputv is an app that runs on every stick and smartphone, so it would be possible to get the whole thing running without the waipu stick

**Reply by PRProd** [reply]
Yes.  It is an app, AND there is a physical Waipu TV Stick as well.  You happened to post this in a discussion about adding support for the physical device, and not as an app shortcut request, which is why I replied the way I did.  Sorry for the misunderstanding.  :smile: 

An app shortcut was added back on Jan 3 2023 as a part of issue #27.  However, the specific request was for only the FireTV app, so that's what was targeted.  What physical device do you own?  Is it an Amazon Fire device?

Feel free to move over to #27 and let me know about your situation.  I will re-open the issue and take care of you over there.

**Reply by AlexZ1402** [reply]
👀🤭 sry

**Reply by PRProd** [reply]
@AlexZ1402 - No worries!  I just took a look at the waipu.tv app again, and I was able to add the shortcut for that app to the Shield, Xiaomi Mi, and Chromecast device support list in the latest release [v2.3.11](https://github.com/PRProd/HA-Firemote/releases/tag/v2.3.11).  I'm not sure if this helps with your issue or not, but I'm hoping it does.

**Comment by PRProd** [comment]
Apple TV won the last round, so after that release the poll over at https://github.com/PRProd/HA-Firemote/discussions/41 has been reset once again. If you're still interested in Firemote support for the Waipu TV Stick, I recommend going back and voting again.
